### ISSUE TITLE
Incorrect value of authorization header in documentation "basic" being used instead of "bearer"

### ISSUE THREAD

**Issue by envy7** [plone/plone.restapi#291]
It is clearly mentioned at http://plonerestapi.readthedocs.io/en/latest/authentication.html#json-web-tokens-jwt that The token can now be used in subsequent requests by including it in the Authorization header with the **Bearer** scheme e.g:
![image](https://cloud.githubusercontent.com/assets/12188993/24207196/54040ab4-0f46-11e7-853d-c87380bb36e3.png)

but everyplace other than here **basic** is being used some examples being:
![image](https://cloud.githubusercontent.com/assets/12188993/24207502/0ac7970c-0f47-11e7-959e-32e8d217f230.png)
![image](https://cloud.githubusercontent.com/assets/12188993/24207561/337d3012-0f47-11e7-888c-a0a39cc92069.png)

which returns an error, I checked with postman
![image](https://cloud.githubusercontent.com/assets/12188993/24207377/ad2e8f42-0f46-11e7-8e14-cfd9e5fead8d.png)

**Comment by lukasgraf** [comment]
@envy7 other places in the documentation use [**HTTP Basic Auth**](https://en.wikipedia.org/wiki/Basic_access_authentication), because that's much easier to set up in tests, and also makes examples much more readable because of short `Authorization` header lines (JSON Web Tokens would be much longer).

That is an entirely different authentication scheme though! You can either use JSON Web Tokens (JWT), and then use the `Bearer` scheme, OR HTTP Basic Auth and then use the `Basic` scheme. If you're trying to send a JWT Token using the `Basic` scheme, that will either lead to unsuccessful authentication at best, or (like in your example) to a decoding error because of mismatching base64 padding.

TL;DR - don't mix the schemes of those two authentication methods. I'll see whether I can add a clarifying comment in the authentication docs.

**Comment by envy7** [comment]
@lukasgraf sorry for the misunderstanding, I should close the issue then ?

**Comment by lukasgraf** [comment]
@envy7 no worries! Yeah, if the issue is resolved for you, let's close it 👍  Just didn't want to close it before being sure that this was the problem you were having 😉
